aea/manager/manager.py

```diff
@@ -300,13 +302,18 @@ def _cleanup(self, only_keys: bool = False) -> None:
                 rmtree(self.working_dir)
 
     def add_project(
-        self, public_id: PublicId, local: bool = True, restore: bool = False
+        self,
+        public_id: PublicId,
+        local: bool = False,
+        remote: bool = False,
+        restore: bool = False,
     ) -> "MultiAgentManager":
         """
         Fetch agent project and all dependencies to working_dir.
 
         :param public_id: the public if of the agent project.
         :param local: whether or not to fetch from local registry.
+        :param remote: whether or not to fetch from remote registry.
```

Comment:
Docstring should make it clear that `local=False` and `remote=False` is `mixed`

Reply:
probably introduce mixed_mode, also for me both false is confusing a bit, I prefer both set to true.